cmd/scw/testdata/test-all-usage-object-bucket-create-usage.golden

```diff
@@ -0,0 +1,22 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Create an Object Storage Bucket with the S3 protocol. The namespace is shared between all S3 users, so its name must be unique.
+
+USAGE:
+  scw object bucket create <name ...> [arg=value ...]
+
+ARGS:
+  name                        The unique name of the bucket
+  [tags.{index}]              List of tags to set on the bucket
```

Comment:
Is it like a key value dictionary for object storage?

Reply:
seem to be